packages/lib/shared/components/navs/NavBar.tsx

```diff
@@ -61,6 +62,12 @@ function NavLinks({
 }) {
   const { linkColorFor } = useNav()
 
+  const handleLinkClick = (analyticsEvent?: string) => {
```

Comment:
Some of the code where we are tracking events is shared with Beets, do we want to track Beets events?